packages/flutter_tools/templates/app/pubspec.yaml.tmpl

```diff
@@ -1,8 +1,6 @@
 name: {{projectName}}
 description: {{description}}
-{{#withPluginHook}}
 publish_to: 'none'
-{{/withPluginHook}}
 {{^withPluginHook}}
```

Comment:
This line should be removed as well.

Reply:
And we may want to add a comment in here about why this line is here and when it should be removed, if ever.